res/graphics/authors.txt

```diff
@@ -161,6 +161,15 @@ parking lane graphics/
 	car4.svg
 	car5.svg
 
+police type graphics/
+    carabinieri.svg                 modified from https://commons.wikimedia.org/wiki/File:Italian_traffic_signs_-_icona_carabinieri.svg, Gigillo83, License: CC-BY-SA
```

Comment:
I would specify also that it is 3.0 version

Reply:
Thanks, I forgot to add it! 😅 Fixing this rn